Updated Cloud Components release notes for 1.0.2 release (#6563)
magento/devdocs: src/cloud/release-notes/mcc-release-notes.md

This release includes the following updates:

> - {:.new}Extended the functionality of the WARM_UP_PAGES environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->

Reviewer: In this topic there are several ways the variable name is emphasized, sometimes as inline code, sometimes bold. Does this one need some emphasis?

> - {:.new}Extended the functionality of the WARM_UP_PAGES environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->

Author: ```suggestion
- {:.new}Extended the functionality of the `WARM_UP_PAGES` environment variable to support cache preloading for specific product pages. See the [post-deploy variables]({{site.baseurl}}/cloud/env/variables-post-deploy.html#warm_up_pages) topic for a detailed feature description.<!--MAGECLOUD-4444-->
```

> -  {:.fix}Fixed an issue caused by an invalid store URL that causes the post-deploy hook to fail when using the `WARM_UP_PAGES` functionality to populate the cache. This issue occurred only when URL rewrites were disabled.<!-- MAGECLOUD-4094 -->

Reviewer: Multiple uses of "cause"... maybe can remove the first one? "Fixed an issue with ..." maybe

> -  {:.fix}Fixed an issue caused by an invalid store URL that causes the post-deploy hook to fail when using the `WARM_UP_PAGES` functionality to populate the cache. This issue occurred only when URL rewrites were disabled.<!-- MAGECLOUD-4094 -->

Author: ```suggestion
-  {:.fix}Fixed an issue where an invalid store URL causes the post-deploy hook to fail when using the `WARM_UP_PAGES` functionality to populate the cache. This issue occurred only when URL rewrites were disabled.<!-- MAGECLOUD-4094 -->
```
